backend/cinema-service/otel-trace/config-otel-staging.properties

```diff
@@ -0,0 +1,6 @@
+otel.service.name=cinema-application-otel
```

Comment:
Remove env props file. There should be only one props file for production which might be `config-otel.properties`